docs/administration-guide/gaffer-deployment/gremlin.md

```diff
@@ -0,0 +1,164 @@
+An example `store.properties` file is given below that will connect to a graph's
+REST API running at `https://localhost:8080/rest/latest`:
+
+```properties
+gaffer.store.class=uk.gov.gchq.gaffer.proxystore.ProxyStore
+# These should be configured to an existing graph deployment
+gaffer.host=localhost
+gaffer.port=8080
+gaffer.context-root=/rest/latest
+```
+
+### Configuring the Gremlin Server
+
+Next we need to configure the Gremlin server, for this the easiest way is to use
+the provided container image from Tinkerpop. To start with simply pull the image
+then we will bind mount in everything we need to make it run with the configured
+Proxy Store and Gaffer Tinkerpop implementation.
+
+```bash
+docker pull tinkerpop/gremlin-server:latest
+```
+
+The first file to create is the `gafferpop.properties`, this is the configuration
+for the Gaffer implementation of Tinkerpop (a.k.a Gafferpop). Most of the set up
+here is for the construction of the Gafferpop Graph instance which we want to
+make run with the `store.properties` we've already configured. An example
+`gaffer.properties` would look like the following:
+
+```properties
+# The Tinkerpop graph class we should use
+gremlin.graph=uk.gov.gchq.gaffer.tinkerpop.GafferPopGraph
+gaffer.graphId=graphProxy
+gaffer.storeproperties=conf/gaffer/store.properties
+gaffer.userId=user01
+```
+
+The second file needed is the configuration for the Gremlin server, this is
+what ties everything together and makes sure the server runs using the Gaffer
+implementation we have configured. A default file is provided in the
+[Tinkerpop repository](https://github.com/apache/tinkerpop/blob/master/gremlin-server/conf/gremlin-server.yaml).
+
+From this file two places need modifying, the first is to change it to use
+our graph configuration file by modifying the `graphs` section like so:
+
+```yaml
+graphs: {
+  graph: conf/gafferpop/gafferpop.properties
+}
+```
+
+The second place is to ensure the Gaffer plugin is loaded for Gremlin which is
+achieved by adding the following to the list of plugins in the `plugins`
+section:
+
+```yaml
+uk.gov.gchq.gaffer.tinkerpop.gremlinplugin.GafferPopGremlinPlugin: {}
+```
+
+!!! tip
+    See the [Tinkerpop docs](https://tinkerpop.apache.org/docs/current/reference/#gremlin-server)
+    for more information on Gremlin server configuration.
+
+### Running the Gremlin Server
+
+After following the previous steps you should now have three custom files
+created which we will bind mount into a `gremlin-server` container. One final
+step however, is to obtain required Gaffer JARs and add them to the container as
+well, there are many different ways to do this the easiest being through maven
+which can use following goal to download all dependencies from a POM:
+
+```bash
+mvn clean dependency:copy-dependencies
+```
+
+Once all JARs are available they can be bind mounted to a path such as,
+`ext/gafferpop/plugin/` in the container to be added to the classpath.
```

Comment:
```suggestion
`ext/gafferpop/plugin/`, in the container to be added to the classpath.
```

Reply:
Not sure this sentence reads too well, could do with a rephrase just to be a little clearer I think